zerver/lib/integrations.py

```diff
@@ -427,7 +427,7 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     WebhookIntegration("greenhouse", ["hr"], display_name="Greenhouse"),
     WebhookIntegration("groove", ["customer-support"], display_name="Groove"),
     WebhookIntegration("harbor", ["deployment", "productivity"], display_name="Harbor"),
-    WebhookIntegration("hellosign", ["productivity", "hr"], display_name="HelloSign"),
+    WebhookIntegration("dropbox sign", ["productivity", "hr"], display_name="Dropbox Sign"),
```

Comment:
I don't think we usually put spaces in the webhook keys in the first parameter? I think those are intended to always match the filename under `zerver/webhooks/`? 

We might need to do something a bit fancy to have the legacy URL work but call the webhook `dropboxsign`.